pkg/services/rendering/http_mode.go

```diff
@@ -194,6 +199,24 @@ func (rs *RenderingService) readFileResponse(ctx context.Context, resp *http.Res
 	return nil
 }
 
+func (rs *RenderingService) getRemotePluginVersionWithRetry(callback func(string, error)) {
+	go func() {
+		var err error
+		for try := uint(0); try < remoteVersionFetchRetries; try++ {
+			version, err := rs.getRemotePluginVersion()
+			if err == nil {
+				callback(version, err)
+				return
+			}
+			rs.log.Info("Couldn't get remote renderer version, retrying", "err", err, "try", try)
```

Comment:
```suggestion
			rs.log.Warn("Couldn't get remote renderer version, retrying", "err", err, "try", try)
```
Maybe? Seems important enough to be more than "Info".